src/providers/bedrock/complete.ts

```diff
@@ -125,6 +125,41 @@ export const BedrockLLamaCompleteConfig: ProviderConfig = {
   },
 };
 
+export const BedrockMistralCompleteConfig: ProviderConfig = {
+  prompt: {
+    param: 'prompt',
+    required: true,
+  },
+  max_tokens: {
+    param: 'max_tokens',
+    default: 20,
+    min: 1,
+  },
+  temperature: {
+    param: 'temperature',
+    default: 0.75,
+    min: 0,
+    max: 5,
+  },
+  top_p: {
```

Comment:
As per their docs, this param is called top_p instead of p for mistral.
```
{
    "prompt": string,
    "max_tokens" : int,
    "stop" : [string],    
    "temperature": float,
    "top_p": float,
    "top_k": int
}
```

Docs: https://docs.aws.amazon.com/bedrock/latest/userguide/model-parameters-mistral.html